hackathonbaobab2021/__init__.py

```diff
@@ -23,7 +22,24 @@ class SportsScheduling(ApplicationCore):
         solvers=list(solvers.keys()),
     )
 
+    @property
+    def instance(self) -> Type[Instance]:
+        return Instance
+
+    @property
+    def solution(self) -> Type[Solution]:
+        return Solution
```

Comment:
We usually do not define this here in the `Application`, why in this case?

Reply:
I needed to have the type hinting in pycharm I think.

Reply:
Ok, I know we define it in the experiment for the same case, if you need it here no problem